eth/api_backend.go

```diff
@@ -41,9 +41,10 @@ import (
 
 // EthAPIBackend implements ethapi.Backend for full nodes
 type EthAPIBackend struct {
-	extRPCEnabled bool
-	eth           *Ethereum
-	gpo           *gasprice.Oracle
+	extRPCEnabled  bool
+	eip155Required bool
```

Comment:
If you define it like this, then the default value is false, and it's not required. Maybe better to define it as `unprotectedAllowed bool` to make it not allow unless explicitly set